### PR TITLE
zh:add tidbmonitor shards doc

### PR DESCRIPTION
**Pull request by mikechengwei**
### What is changed, added, or deleted? (Required)

add tidbmonitor shards doc.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]

File: zh/enable-monitor-shards.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+# 开启 TidbMonitor 分片功能
+
+本文档介绍如何使用 TidbMonitor 分片功能。
+
+## 功能介绍
+
+TidbMonitor 负责单个或者多个 TiDB 集群的监控数据采集。当监控数据量很大的时候，单点计算能力会达到瓶颈。可以采用 Prometheus [Modulus](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) 分片功能，对 `__address__` 做 `hashmod`，打散 `Targets` 到多个 TidbMonitor 节点上。
+
+这种方式需要支持数据聚合方案，我们推荐采用 [Thanos](https://thanos.io/tip/thanos/design.md/) 方案。
```
Comment (qiancai) [inline zh/enable-monitor-shards.md:14]:
```suggestion

## 前提条件

TidbMonitor 分片功能需要支持数据聚合方案。推荐采用 [Thanos](https://thanos.io/tip/thanos/design.md/) 方案。
```

Comment (qiancai) [reply zh/enable-monitor-shards.md:14]: @mikechengwei 请问TidbMonitor 分片功能是否必须配置数据聚合方案呢？

File: zh/enable-monitor-shards.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    version: v5.2.1
+  reloader:
+    baseImage: pingcap/tidb-monitor-reloader
+    version: v1.0.1
+  imagePullPolicy: IfNotPresent
+```
+
+> **注意：**
+>
+> Pod 数量取决于 replica 和 shard 的乘积。当 replica 为 2 个副本，shard 为 2 个分片，就会产生四个 TidbMonitor Pod实例。
```
Comment (qiancai) [inline zh/enable-monitor-shards.md:44]:
```suggestion
> TidbMonitor 对应的 Pod 数量取决于 replica 和 shard 的乘积。例如，当 replica 为 1 个副本，shard 为 2 个分片时，TiDB Operator 会产生 2 个 TidbMonitor Pod实例。
```

Comment (qiancai) [reply zh/enable-monitor-shards.md:44]: 为了方便用户理解，建议数量设置与上述配置文件保持一致

File: zh/enable-monitor-shards.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+summary: 如何使用 TidbMonitor 分片功能
+---
+
+# 开启 TidbMonitor 分片功能
+
+本文档介绍如何使用 TidbMonitor 分片功能。
+
+## 功能介绍
+
+TidbMonitor 负责单个或者多个 TiDB 集群的监控数据采集。当监控数据量很大的时候，单点计算能力会达到瓶颈。可以采用 Prometheus [Modulus](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) 分片功能，对 `__address__` 做 `hashmod`，打散 `Targets` 到多个 TidbMonitor 节点上。
```
Comment (qiancai) [inline zh/enable-monitor-shards.md:12]: 这里的 `Targets` 是什么呢？是被监控的数据吗

**Review by qiancai — COMMENTED** [review]
@mikechengwei

File: zh/enable-monitor-shards.md (hunk 1 above)
Comment (mikechengwei) [reply zh/enable-monitor-shards.md:14]: 是的，必须

File: zh/enable-monitor-shards.md (hunk 3 above)
Comment (mikechengwei) [reply zh/enable-monitor-shards.md:12]: 被监控的节点。

Comment (mikechengwei) [reply zh/enable-monitor-shards.md:12]: 被监控的节点，关键字叫 Targets

File: zh/enable-monitor-shards.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  reloader:
+    baseImage: pingcap/tidb-monitor-reloader
+    version: v1.0.1
+  imagePullPolicy: IfNotPresent
+```
+
+> **注意：**
+>
+> TidbMonitor 对应的 Pod 数量取决于 replica 和 shard 的乘积。例如，当 replica 为 1 个副本，shard 为 2 个分片时，TiDB Operator 会产生 2 个 TidbMonitor Pod实例。
+> shard 变更后，Targets 会重新分配，但是原本在节点上的监控数据不会重新分配。
```
Comment (qiancai) [inline zh/enable-monitor-shards.md:47]:
```suggestion
> - TidbMonitor 对应的 Pod 实例数量取决于 `replica` 和 `shard` 的乘积。例如，当 `replica` 为 1 个副本，`shard` 为 2 个分片时，TiDB Operator 将产生 2 个 TidbMonitor Pod 实例。
> - `shard` 变更后，`Targets` 会重新分配，但是原本在节点上的监控数据不会重新分配。
```

File: zh/enable-monitor-shards.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+可以参考 [Example](https://github.com/pingcap/tidb-operator/tree/master/examples/monitor-shards)。
```
Comment (qiancai) [inline zh/enable-monitor-shards.md:49]:
```suggestion
可以参考 [分片示例](https://github.com/pingcap/tidb-operator/tree/master/examples/monitor-shards)。
```

File: zh/enable-monitor-shards.md (hunk 3 above)
Comment (qiancai) [reply zh/enable-monitor-shards.md:12]:
```suggestion
TidbMonitor 负责单个或者多个 TiDB 集群的监控数据采集。当监控数据量很大的时候，单点计算能力会达到瓶颈。此时，你可以采用 Prometheus [Modulus](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) 分片功能，对 `__address__` 做 `hashmod`，将对目标节点 （关键字为 `Targets`）的监控打散到多个 TidbMonitor 节点上进行。
```

File: zh/enable-monitor-shards.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    version: v5.2.1
+  reloader:
+    baseImage: pingcap/tidb-monitor-reloader
+    version: v1.0.1
+  imagePullPolicy: IfNotPresent
+```
+
+> **注意：**
+>
+> - TidbMonitor 对应的 Pod 实例数量取决于 `replica` 和 `shard` 的乘积。例如，当 `replica` 为 1 个副本，`shard` 为 2 个分片时，TiDB Operator 将产生 2 个 TidbMonitor Pod 实例。
```
Comment (DanielZhangQD) [inline zh/enable-monitor-shards.md:46]:
```suggestion
> - TidbMonitor 对应的 Pod 实例数量取决于 `replicas` 和 `shards` 的乘积。例如，当 `replicas` 为 1 个副本，`shards` 为 2 个分片时，TiDB Operator 将产生 2 个 TidbMonitor Pod 实例。
```

File: zh/enable-monitor-shards.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+> - `shard` 变更后，`Targets` 会重新分配，但是原本在节点上的监控数据不会重新分配。
```
Comment (DanielZhangQD) [inline zh/enable-monitor-shards.md:47]:
```suggestion
> - `shards` 变更后，`Targets` 会重新分配，但是原本在节点上的监控数据不会重新分配。
```

File: zh/enable-monitor-shards.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+summary: 如何使用 TidbMonitor 分片功能
+---
+
+# 开启 TidbMonitor 分片功能
+
+本文档介绍如何使用 TidbMonitor 分片功能。
+
+## 功能介绍
+
+TidbMonitor 负责单个或者多个 TiDB 集群的监控数据采集。当监控数据量很大的时候，单点计算能力会达到瓶颈。此时，你可以采用 Prometheus [Modulus](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) 分片功能，对 `__address__` 做 `hashmod`，将对目标节点（关键字为 `Targets`）的监控打散到多个 TidbMonitor 节点上进行。
```
Comment (DanielZhangQD) [inline zh/enable-monitor-shards.md:12]:
```suggestion
TidbMonitor 负责单个或者多个 TiDB 集群的监控数据采集。当监控数据量很大的时候，单点计算能力会达到瓶颈。此时，你可以采用 Prometheus [Modulus](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) 分片功能，对 `__address__` 做 `hashmod`，将多个目标（关键字为 `Targets`）的监控打散到多个 TidbMonitor Pod 上。
```

File: zh/enable-monitor-shards.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+## 前提条件
+
+TidbMonitor 分片功能需要支持数据聚合方案。推荐采用 [Thanos](https://thanos.io/tip/thanos/design.md/) 方案。
+
```
Comment (DanielZhangQD) [inline zh/enable-monitor-shards.md:17]:
```suggestion
TidbMonitor 分片功能需要采用数据聚合方案，推荐使用 [Thanos](https://thanos.io/tip/thanos/design.md/) 方案。

```

File: zh/enable-monitor-shards.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+## 如何开启分片功能
+
+开启分片功能，需要指定 `shard` 字段，示例如下:
```
Comment (DanielZhangQD) [inline zh/enable-monitor-shards.md:20]:
```suggestion
开启分片功能，需要指定 `shards` 字段，示例如下:
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**